pyfolio/risk.py

```diff
@@ -432,22 +435,26 @@ def compute_volume_exposures(shares_held, volumes, percentile):
     shorted_frac = shares_shorted.divide(volumes)
     grossed_frac = shares_grossed.divide(volumes)
 
-    # NOTE: To work around a bug with nan-handling in pandas 0.18,
-    #       drop all-nan rows before the call to `quantile`, and then
-    #       restore them by reindexing the final result. This is fixed
-    #       in pandas 0.19.
-    longed_threshold = 100 * longed_frac.dropna(how='all').quantile(
-        percentile,
+    # NOTE: To work around a bug in `quantile` with nan-handling in
+    #       pandas 0.18, use np.nanpercentile by applying to each row of
+    #       the dataframe. This is fixed in pandas 0.19.
+    #
+    # longed_threshold = 100*longed_frac.quantile(percentile, axis='columns')
```

Comment:
cruft?

Reply:
I was just leaving these as a reference for when we no longer need to support 0.18, does that sound alright?

Reply:
yup sounds good